FlymFork/src/main/res/values-de/strings.xml

```diff
@@ -23,24 +22,24 @@
  THE SOFTWARE.
 -->
 <resources>
-
     <!-- GENERAL -->
-    <string name="about_flym">Über app / Help / Contact developer </string>
-    <string name="welcome_title">Wilkommen in Flym. Willst du einen Feed hinzufügen?</string>
+    <string name="app_name">Flym Fork</string>
+    <string name="about_flym">Info - Hilfe - Support</string>
+    <string name="welcome_title">Willkommen bei Flym Fork. Möchten Sie einen Feed hinzufügen?</string>
```

Comment:
Change app name to Handy News Reader please

Reply:
Okay, I will do it. I wonder why the english strings.xml says "Flym Fork"?

Reply:
Done...